assets/themes/zeppelin/css/style.css

```diff
@@ -88,32 +99,41 @@ body {
 .navbar {
   background: #3071a9;
   border-bottom: 0px;
-  height: 50px;
+  height: 60px;
+  -webkit-box-shadow: 0 8px 6px -6px #999;
```

Comment:
what you say is this line( https://github.com/apache/zeppelin/pull/2371/files#diff-2749d9139fea273c8b492b39bc12eb7dR87)? it looks same color.